libres/lib/enkf/time_map.cpp

```diff
@@ -350,42 +360,51 @@ bool time_map_update(time_map_type *map, int step, time_t time) {
 }
 
 bool time_map_try_update(time_map_type *map, int step, time_t time) {
-    bool updateOK;
     time_map_assert_writable(map);
+    std::string error;
     pthread_rwlock_wrlock(&map->rw_lock);
-    { updateOK = time_map_update__(map, step, time); }
+    error = time_map_update__(map, step, time);
```

Comment:
Any reason not to do `std::string error` here?

Reply:
No, was an oversight, fixed, thanks :+1: